tests/Console/ModelsCommand/AbstractModelsCommand.php

```diff
@@ -44,6 +44,9 @@ protected function getEnvironmentSetUp($app)
             'prefix' => '',
         ]);
 
+        // Set the default models helper filename
+        $config->set('ide-helper.models_filename', '_ide_helper_models.php');
```

Comment:
I think for the same reason I mention this in the app code, this actually shouldn't be necessary

Reply:
I thought this was weird too, but when I dump `$config->get('ide-helper')` (the entire array), I only get the values that were set explicitly. That's why I added this value as well.

Reply:
Did you clear your config cache? Or did you override the config. A default value should be used when calling it.

Reply:
I don't know what went wrong, but the tests seem to pass without adding the value explicitly now, so I've removed these lines.